templates/_schemas/details/default.twig

```diff
@@ -2,14 +2,29 @@
 
 
 
+{% set graph = create('Spatie\\SchemaOrg\\Graph') %}
+{% set seomateMetaData = craft.seomate.getMeta({
+  config: { includeSitenameInTitle: false },
+}|merge(seomate ?? {}, true)) %}
 {% set entryImages = entryImages ?? craft.assets.relatedTo(entry).kind('image').all() %}
 
 {% set defaultSchemaProps = {
-  title: entry.title,
+  title: seomateMetaData.meta.title ?? entry.title,
+  description: entry.description ?? seomateMetaData.meta.description,
```

Comment:
We should not assume the presence of any custom fields like `entry.description` in the boilerplate. Those are project-specific. What if one project we don’t have a field called `description`, or what if it is present but not on all sections?

Reply:
Make sense. Removed `description`.

Reply:
Why are we removing the `description` meta data itself? We should not use `entry.description`, but we can still use the SEO description value as the JSON-LD description as well, right?

Reply:
Done.